packages/core/admin/admin/src/components/ReleaseNotification/index.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { useAppInfos, useNotification } from '@strapi/helper-plugin';
+
+const showUpdateNotif = !JSON.parse(localStorage.getItem('STRAPI_UPDATE_NOTIF'));
+
+const ReleaseNotification = () => {
```

Comment:
When something uses hooks and returns nothing, it's a good sign it might become a hook 😛

Reply:
Ideally it should be a hook and we should use it in the Admin component. However, since the Admin is still a class component we cannot use a hook at the moment.

WDYT about adding a comment in this component and switch it to a hook once we have reworked the Admin page?

Reply:
LGTM 😊

Reply:
Is it not used in the PluginsInitializer functional component? 🤔